src/main/resources/db/migration/V051__Add_foreign_key_indexes.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX scannable_items_envelope_id_idx ON scannable_items (envelope_id);
+CREATE INDEX non_scannable_items_envelope_id_idx ON non_scannable_items (envelope_id);
```

Comment:
although we don't have non scannable items data currently but worth adding it now so that when it starts coming we don't have to change anything